tests/unit/test_ops.py

```diff
@@ -680,6 +685,169 @@ def test_integration_multiple_ts():
         )
 
 
+@pytest.mark.parametrize("out_var", (None, "new out var"))
+def test_integration_sum(out_var):
+    dat = [1, 2, 3]
+    start = get_single_ts(data=dat, index=[2020, 2021, 2022], unit="GtC / yr")
+
+    res = start.integrate(out_var=out_var, method="sum")
+
+    if out_var is None:
+        exp_var = ("Cumulative " + start["variable"]).values
+    else:
+        exp_var = out_var
+
+    exp = get_single_ts(
+        data=np.array([1, 3, 6]),
+        index=[2020, 2021, 2022],
```

Comment:
Should 2020 cumulative emissions be 1 or should 2021 cumulative emissions be 1 (the emissions haven't actually happened in 2020, they're only fully realised in 2021...). I'm not sure which convention is more confusing.

Reply:
For cumulative emissions case specifically, i think it should represent the cumulative sum up to and including a given year. One use case that would be handy to support is `cum_emis.filter(year=2100)` where you're summing from (and including) the reference year up to and including 2100. 

I'm not sure about the `trapz` method. Are there any particular use cases where integrating concentrations or any other piecewise linear timeseries would be of interest?

Reply:
Ok makes sense. Can you add a note to the docs about the result being 'up to and including'?

Re `trapz`, I would just copy the same convention as `sum`so the result is always sort of back-filled (i.e. the last timestep always ends up being nan rather than the first one always being zero)

Reply:
Added docs to `cumsum` and `cumtrapz` explaining the difference